src/styles/globals.css

```diff
@@ -1,6 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@import "./dsfr.css";
```

Comment:
nit: should import be placed at the top of the file?

Reply:
oh good catch, too used to scss… I'll actually rework the CSS imports to make sure  everything is included in correct order, I don't want classes to be included before reset/base css to prevent specificity issues